authors/settings.py

```diff
@@ -190,7 +190,10 @@
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'authors.apps.authentication.backends.JWTAuthentication',
     ),
+
 }
+PAGE_SIZE = os.getenv("PAGE_SIZE")
```

Comment:
Hey @JoshuaKodhe great way of implementing pagination, allows us to quickly change page size from our environment variable